projects/dynamicforms/src/components/datepicker/datepicker.component.html

```diff
@@ -13,7 +13,7 @@
         placeholder="{{ getDateFormat() }}"
         [formControlName]="config.name"
         [attr.name]="config.name"
-        [attr.readonly]="config.readonly"
+        [attr.readonly]="config.readonly ? config.readonly : null"
```

Comment:
It's enough to have **readonly** attribute with any value to make the form element mutable.